src/management-system-v2/components/properties-panel.tsx

```diff
@@ -23,19 +22,20 @@ import CustomPropertySection from './custom-property-section';
 import MilestoneSelectionSection from './milestone-selection-section';
 import ResizableElement, { ResizableElementRefType } from './ResizableElement';
 import CollapsibleCard from './collapsible-card';
+import DescriptionSection from './description-section';
+import ImageSelectionSection from './image-selection-section';
```

Comment:
This should be "image-selection". Currently getting an error when trying to start the MS